app/src/main/java/com/sdp13epfl2021/projmag/model/ProjectFilter.kt

```diff
@@ -75,4 +83,17 @@ class ProjectFilter(
         isAppliedProject = appCheck
         return this
     }
+
+    /**
+     * Function that will check if the project is contained in the favorites
+     *
+     * @param favCheck : Function that will take a project and indicate if it's in the favorite list
+     * @return returns the filter
```

Comment:
return is redundant

Reply:
You could use `@return this filter` like the previous function